package.json

```diff
@@ -41,6 +41,7 @@
     ]
   },
   "resolutions": {
+    "**/@graphql-tools/url-loader/**/ws": "7.4.6",
```

Comment:
Maybe this, to continue receiving patch updates? :)

```suggestion
    "**/@graphql-tools/url-loader/**/ws": "^7.4.6",
```

Reply:
Makes sense!

Reply:
Done :)